buildSrc/src/main/kotlin/software/amazon/app/platform/gradle/buildsrc/KmpPlugin.kt

```diff
@@ -132,6 +132,17 @@ public open class KmpPlugin : Plugin<Project> {
 
         kmpExtension.sourceSets.named("apple$suffix").configure { it.dependsOn(appleAndDesktop) }
         kmpExtension.sourceSets.named("desktop$suffix").configure { it.dependsOn(appleAndDesktop) }
+
+        val noWasmJs = kmpExtension.sourceSets.create("noWasmJs$suffix")
```

Comment:
This source set is needed for the `Waiter` class, because `runBlocking` doesn't exist JS.